core/testing/edu/asu/c3simulator/testing/stubs/ProductTestingStub.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * 
+ */
+package edu.asu.c3simulator.testing.stubs;
+
+import java.text.DecimalFormat;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Product;
+
+/**
+ * Stub to test the product interface
+ * @author Justin
+ *
+ */
+public class ProductTestingStub implements Product
+{
+	private String name, laborer;
+	private float productionCost, materialCost, distributionCost, sellingPrice;
+	private int efficiency;
+	/* (non-Javadoc)
+	 * @see edu.asu.c3simulator.simulation.Product#getProduct()
+	 */
```

Comment:
Get rid of these non-javadocs, unless they add something of value (in which case they should be javadoc comments)